### PR TITLE
don't set 'use-credentials'

### PR DESCRIPTION
**Pull request by aliceriot**
This option was set during development / testing and snuck into the merged PR

### CODE REVIEW

File: webpack.config.prod.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ module.exports = Object.assign(prodConfig, {
     path: path.resolve('./static/bundles/'),
     filename: "[name]-[chunkhash].js",
     chunkFilename: "[id]-[chunkhash].js",
-    crossOriginLoading: "use-credentials",
```
Comment (rhysyngsun) [inline webpack.config.prod.js]: I think we actually need to set this to "anonymous" because the default is `false`, which disables it entirely.

Comment (aliceriot) [inline webpack.config.prod.js]: cool, will do

**Comment by rhysyngsun** [comment]
Assuming the tests pass :+1:

**Comment by aliceriot** [comment]
Do you think we should confirm it works ok with cloudfront first? Looking into setting it up on the PR build to test.

**Comment by aliceriot** [comment]
alright, so the cloudfront stuff is set up on the PR build now, trying to get that working to confirm the fix is good

**Comment by aliceriot** [comment]
@noisecapella So the `CLOUDFRONT_DIST` env var is set, and it seems like django is picking up on that, but none of the static assets are being fetched. Do you know if there's any other configuration I need to do for it?

**Comment by aliceriot** [comment]
It seems like it is working now! I think before an issue was that heroku for some reason was not doing the python portion of the build, not sure.

But @rhysyngsun do you want to confirm? I think just confirm that if you go to the PR review app you can use the learner search page.

**Comment by rhysyngsun** [comment]
:+1: Confirmed the js bundles are loading without error from CF on the PR build.

**Pull request merged**